README.md

```diff
@@ -0,0 +1,9 @@
+# code.angularjs.org
+
+This repo contains the content served at [code.angularjs.org](https://code.angularjs.org).
```

Comment:
this readme will be seen primarily by the visitors of code.angularjs.org so I'd made it as README.txt and tailor the message to the code.angularjs.org visitors rather than visitors of this source repo.